services/httperrors/httperrors_test.go

```diff
@@ -3,7 +3,7 @@ package httperrors_test
 import (
```

Comment:
For all tests after some research please use
https://pkg.go.dev/github.com/stretchr/testify/assert
to make assertion in you unit test, it's more readable